app/uk/gov/hmrc/cdsreimbursementclaimfrontend/views/claims/enter_duplicate_mrn.scala.html

```diff
@@ -51,8 +51,8 @@
 
         @inputText(
             form = form,
-            id = key,
-            name = key,
+            id = "enter-duplicate-mrn-or-entry-number",
```

Comment:
you need to pass in the `key` method here - if the one in the file is incorrect then something is not correct here: either the key is incorrect or the template doesn't fit the purpose.

Reply:
This is the reason I did the whole refactoring.
there are 5 templates for the MRN page. Each had a separate id+name to submit. And when Serg selected the proper form and feature switch, he selected the wrong template/form and you could not pass the page with any MRN or Entry Number. So to simplify, I moved all Duplicate MRN pages (there are 3 of them) to submit: "enter-duplicate-mrn-or-entry-number". That way I could remove the complicated - and erroneous - logic for the form selection.

Reply:
The problem with this approach is that it will make it difficult to re-use the template when the other journeys are built in.